Chart: Update axis grid topics (#7650)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/1 Configuration/commonAxisSettings/grid/grid.md

Configures the grid.

---

> Grid lines can be considered extensions of [ticks](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/tick '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/tick/').

Reviewer: ```suggestion
Gridlines can be considered extensions of [ticks](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/tick '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/tick/').
```

                [visible]="true"
                color="blue"
                [opacity]="0.25"
                [width]="2"
            ></dxo-grid>
        </dxo-common-axis-settings>
    </dx-chart>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxChart ... >
            <DxCommonAxisSettings>
                <DxGrid
                    :visible="true"
                    color="blue"
                    :opacity="0.25"
                    :width="2"
                />
            </DxCommonAxisSettings>
        </DxChart>
    </template>

    <script setup lang="ts">
    import { DxChart, DxCommonAxisSettings, DxGrid } from 'devextreme-vue/chart';

    </script>

##### React

    <!-- tab: App.tsx -->
    import { Chart, CommonAxisSettings, Grid } from 'devextreme-react/chart';

    function App() {
        return (
            <Chart ... >
                <CommonAxisSettings>
                    <Grid
                        visible={true}
                        color="blue"
                        opacity={0.25}
                        width={2}
                    />
                </CommonAxisSettings>
            </Chart>
        )
    }

---

> The **commonAxisSettings**.[grid](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/grid '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/grid/') object specifies common settings for all grid lines in the chart. To configure only those grid lines that descend from a particular axis, use the following objects.

Reviewer: ```suggestion
The **commonAxisSettings**.[grid](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/grid '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/grid/') object specifies common settings for all gridlines on a chart. To configure only those grid lines that descend from an axis of your choice, use the following objects.
```

Author: ```suggestion
The **commonAxisSettings**.[grid](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/commonAxisSettings/grid '/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/grid/') object specifies common settings for all gridlines on a chart. To configure only those grid lines that descend from a particular axis, use the following objects.
```
